plutus-core/testlib/PlutusCore/Test.hs

```diff
@@ -537,9 +538,9 @@ test_scopingGood ::
   -- | The runner of the pass.
   (t NameAnn -> TPLC.Quote (t NameAnn)) ->
   TestTree
-test_scopingGood pass gen bindRem preren run =
+test_scopingGood n pass gen bindRem preren run =
   testPropertyNamed (pass ++ " does not break scoping and global uniqueness") "test_scopingGood" $
-    prop_scopingFor gen bindRem preren run
+    withAtLeastTests n (prop_scopingFor gen bindRem preren run)
```

Comment:
You can pass this via the CLI. I think you could just put this in your GHA and avoid the whole additional test suite, also it will run all the property tests in the test suite with 1000 tests.
```
cd plutus-core # it needs to run in the package directory
nix run .#hydraJobs.x86_64-linux.ghc92.components.tests.plutus-core.plutus-ir-tests -- --quickcheck-tests 1000
```
(I checked this, it works)

Reply:
When running the whole `plutus-core:plutus-core-test` with `--hedgehog-tests 100000`, the GH action got killed after about one hour ([attempt1](https://github.com/input-output-hk/plutus/actions/runs/5943963244/attempts/1), [attempt2](https://github.com/input-output-hk/plutus/actions/runs/5943963244/attempts/2)). Unclear whether they are OOM kills, but GH-hosted runners do have relatively tight resources. I lowered the number of test cases from 100k to 10k.